src/UI/Internal/FiltersHeaders.elm

```diff
@@ -125,6 +144,8 @@ headerAttrs isApplied =
             else
                 [ Background.color Palette.gray.lightest
                 , Font.color Palette.primary.darkest
+                , Element.mouseOver
+                    [ Background.color Palette.primary.lightest ]
```

Comment:
I would use the same smooth transition we have for the buttons, but it's up to you

Reply:
You're right, applying it...